app.js

```diff
@@ -1,16 +1,139 @@
 /* Imports */
+import { getRandomItem } from './utils.js';
 
 /* State */
+let gameState = ''; // 'guess' or 'results'
+let userPlay = ''; //'rock' or 'paper' or 'scissors'
+let computerPlay = ''; //'rock' or 'paper' or 'scissors'
+let result = ''; // 'win' or 'lose' or 'draw'
+
+let wins = 0;
+let losses = 0;
+let draws = 0;
+let totalPlays = 0;
+
+// probability array
+const choices = ['rock', 'paper', 'scissors'];
 
 /* Actions */
-function load() {}
+function loadPage() {
+    displayGuess();
+    displayResults();
+    displayScorecard();
+}
+
+/* User Play */
+// get DOM
+const throwRock = document.getElementById('throw-rock-button');
+const throwPaper = document.getElementById('throw-paper-button');
+const throwScissors = document.getElementById('throw-scissors-button');
+const userPlaySection = document.getElementById('user-play-section');
+const resultDisplay = document.getElementById('result-display');
+
+// display
+function displayGuess() {
+    resultsSection.classList.add('hidden');
+}
+
+function playGame(userGuess) {
+    gameState = 'results';
+    userPlay = userGuess;
+    computerPlay = getRandomItem(choices);
+    totalPlays++;
+
+    userPlaySection.classList.add('hidden');
+    resultsSection.classList.remove('hidden');
 
-/* Components */
+    if (userPlay === computerPlay) {
+        result = 'draw';
+        draws++;
+        resultDisplay.textContent = 'A draw!';
+    }
+    if (userPlay === 'rock' && computerPlay === 'scissors') {
+        result = 'win';
+        wins++;
+        resultDisplay.textContent = 'You won!';
+    }
+    if (userPlay === 'rock' && computerPlay === 'paper') {
+        result = 'lose';
+        losses++;
+        resultDisplay.textContent = 'You lost.';
+    }
+    if (userPlay === 'paper' && computerPlay === 'rock') {
+        result = 'win';
+        wins++;
+        resultDisplay.textContent = 'You won!';
+    }
+    if (userPlay === 'paper' && computerPlay === 'scissors') {
+        result = 'lose';
+        losses++;
+        resultDisplay.textContent = 'You lost.';
+    }
+    if (userPlay === 'scissors' && computerPlay === 'rock') {
+        result = 'lose';
+        losses++;
+        resultDisplay.textContent = 'You lost.';
+    }
+    if (userPlay === 'scissors' && computerPlay === 'paper') {
+        result = 'win';
+        wins++;
+        resultDisplay.textContent = 'You win.';
+    }
+    console.log('user threw', userGuess);
+    console.log('computer threw', computerPlay);
+    console.log('result is', result);
+    console.log('total plays', totalPlays);
+    console.log(resultDisplay.textContent);
```

Comment:
Great job seeing what is happening behind the scenes with the console.logs but take them out when you are done! It just keeps your code clean :)

Reply:
Will do!